src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/DataElement.cs

```diff
@@ -72,6 +72,13 @@ public class DataElement : ChangableElement
         [JsonProperty(PropertyName = "refs")]
         public List<Guid> Refs { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the element is read.
+        /// </summary>
+        [JsonProperty(PropertyName = "isRead")]
+        [DefaultValue(true)]
```

Comment:
using the default value property I guess we could even assign it as null? Could that be a better way of communicating that the read status isn't tracked at all, rather than true which communicates that something might be read even if it isn't although the app owner doesn't really care?

Reply:
If we want the property to be `null` if no value has been given, then any default value behavior is unnecessary. The default value of `bool?` is already `null`. In this case any logic using the property for anything will need to check if it `.HasValue` every time.

If we simply want default behavior of any logic to be as if the value is `true`, then we could use default value to set it to `true` initially. My idea here was to have a backing field and in the property get method we could have a simple check. `If null return true`. Serialization will then cause it to be saved as `true`. Never as `null`. Unless I'm mistaken.